jobserver/settings.py

```diff
@@ -239,6 +239,7 @@ def immutable_file_test(path, url):
 SOCIAL_AUTH_GITHUB_KEY = env.str("SOCIAL_AUTH_GITHUB_KEY")
 SOCIAL_AUTH_GITHUB_SECRET = env.str("SOCIAL_AUTH_GITHUB_SECRET")
 SOCIAL_AUTH_GITHUB_SCOPE = ["user:email"]
+CONTROLLER_API_ENDPOINT = env.str("CONTROLLER_API_ENDPOINT")
```

Comment:
Should this be `RAP_API_ENDPOINT`?  Clients of the RAP API shouldn't need to know about controller/agent terms